packages/realm/src/Listeners.ts

```diff
@@ -54,3 +56,43 @@ export class Listeners<CallbackType> {
     this.listeners.clear();
   }
 }
+
+/** @internal */
+type BindingListenerCallback = (r: Realm, name: string, schema?: Realm.ObjectSchema[]) => void;
+/** @internal */
+export class BindingListeners {
```

Comment:
```suggestion
export class RealmListeners {
```

I'd suggest this class gets a file of its own 👍